teps/0096-pipelines-v1-api.md

```diff
@@ -239,6 +239,7 @@ This policy should be updated to include Tekton metrics as part of the API. No o
 - Fix [pain points](https://github.com/tektoncd/pipeline/issues/3792) associated with TaskRun and PipelineRun Status.
 - Rename PipelineRun's `spec.taskRunSpecs.taskServiceAccountName` to `spec.taskRunSpecs.serviceAccountName`.
 - Rename PipelineRun's `spec.taskRunSpecs.taskPodTemplate` to `spec.taskRunSpecs.podTemplate`.
+- Rename `pipelineRun.spec.taskRunSpecs[].stepOverrides` to `pipelineRun.spec.taskRunSpecs[].stepSpecs` and `pipelineRun.spec.taskRunSpecs[].sidecarOverrides` to `pipelineRun.spec.taskRunSpecs[].sidecarSpecs`,
```

Comment:
Thank you for making these changes! could you please just add these fields to the list on line 252-256?